src/NUnitFramework/framework/Internal/Tests/ParameterizedFixtureSuite.cs

```diff
@@ -42,6 +42,16 @@ public ParameterizedFixtureSuite(ITypeInfo typeInfo) : base(typeInfo.Namespace,
             _genericFixture = typeInfo.ContainsGenericParameters;
         }
 
+        /// <summary>
+        /// Copy constructor style to create a filtered copy of the given parameterized fixture suite
+        /// </summary>
+        /// <param name="suite">Parameterized Fixture Suite to copy</param>
+        /// <param name="filter">Filter to be applied</param>
```

Comment:
```suggestion
        /// <param name="filter">Determines which descendants are copied.</param>
```

(Same comment for all constructors _and_ Copy methods.)